scripts/readme.js

```diff
@@ -0,0 +1,121 @@
+  const { stdout: packageList } = await execa('lerna', ['list', '-a', '--json']);
+  return JSON.parse(packageList)
+    .map(pkg => ({
+      ...pkg,
+      location: path.relative(process.cwd(), pkg.location),
+    }));
+};
+
+const filterPackages = (
+  pkgs,
+  { publicPackage = true, privatePackage = false, packagePath },
+) => {
+  let filteredPackages = [...pkgs];
+  if (!publicPackage) {
+    filteredPackages = filteredPackages.filter(pkg => pkg.private);
+  }
+  if (!privatePackage) {
+    filteredPackages = filteredPackages.filter(pkg => !pkg.private);
+  }
+  if (packagePath) {
+    filteredPackages = filteredPackages.filter(pkg => pkg.location.startsWith(packagePath));
+  }
+
+  return filteredPackages;
+};
+
+const updateReadme = async () => {
+  const packages = await getPackages();
+  const markdownPath = path.join(process.cwd(), 'README.md');
+  markdownMagic(
+    markdownPath,
+    {
+      transforms: {
+        /*
+          Match :
+            <!-- AUTO-GENERATED-CONTENT:START (PACKAGES:showBadges=false&name=module) -->
+              <!-- CONTENT WILL BE GENERATED HERE -->
+            <!-- AUTO-GENERATED-CONTENT:END *-->
+          Default Options: {
+            packagePath: '',
+            publicPackage: true,
+            privatePackage: false,
+            showBadges: true,
+            name: 'package',
+          }
+        */
+        PACKAGES: (content, _options = {}) => {
+          const options = {
+            packagePath: _options.packagePath || false,
+            publicPackage: (_options.publicPackage || 'true') === 'true',
+            privatePackage: (_options.privatePackage || 'false') === 'true',
+            showBadges: (_options.showBadges || 'true') === 'true',
+            name: _options.name || 'packages',
+          };
+
+          const packageList = filterPackages(packages, options);
+
+          if (packageList.length === 0) {
+            return '';
+          }
+
+          const headers = [
+            `# Availables ${options.name}`,
+            '',
+          ];
+
+          let cols = ['App'];
+          if (options.showBadges) {
+            cols = [
+              ...cols,
+              'Version',
+              'Dependencies',
+              'Dev dependencies',
+              'Peer dependencies',
+            ];
+          }
+          const separators = cols.map(col => '-'.repeat(col.length));
+          cols.push('Changelog');
+          separators.push(':-------:');
+
+          headers.push(`| ${cols.join(' | ')} |`);
+          headers.push(`| ${separators.join(' | ')} |`);
+          headers.push('');
+
+
+          const lines = packageList.map(({ name, location }) => {
+            const line = [
+              `[${name}](https://github.com/ovh-ux/manager/tree/master/${location})`,
+            ];
+
+            if (options.showBadges) {
+              line.push(`[![npm version](https://badgen.net/npm/v/${name})](https://www.npmjs.com/package/${name})`);
```

Comment:
You can use multi-line strings inside template literals.

Reply:
Yes, I push too early.
I want to set `showBadges=version|deps|devDeps|peerDeps` to be more agile : 
If you set `showBadges=version` you will have only `version` badge.